serve/mlc_serve/engine/engine_common.py

```diff
@@ -118,8 +118,6 @@ def detokenize_incrementally(
         prefix_begin_offset = generation_sequence.prefix_begin_offset
         prefix_end_offset = generation_sequence.prefix_end_offset
 
-    assert tokenizer.is_fast
```

Comment:
Qwen tokenizer fails this assert.

Reply:
Can we leave a comment somewhere? Just for knowledge sharing.